contracts/swap/src/testing/swap_tests.rs

```diff
@@ -36,7 +36,10 @@ fn it_reverts_if_atomic_fee_multiplier_query_fails() {
         &deps.as_mut_deps().as_ref(),
         &env,
         &MarketId::unchecked(TEST_MARKET_ID_1.to_string()),
-        FPCoin::from(Coin::new(1000000000000000000u128, "eth".to_string())),
+        SwapEstimationAmount::InputQuantity(FPCoin::from(Coin::new(
+            1000000000000000000u128,
```

Comment:
pls never do that again, use `str_coin`.

Reply:
not my code, complain to Antoni please 😅

Reply:
not my code, complain to Antoni please 😅